dokku.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eu
+
+cd /tmp && wget https://github.com/casey/just/releases/download/1.40.0/just-1.40.0-x86_64-unknown-linux-musl.tar.gz && tar -xzf just-1.40.0-x86_64-unknown-linux-musl.tar.gz && chmod 555 just
+sudo mv /tmp/just /usr/bin/
+
+cd /workspaces/opencodelists
+cp dotenv-sample .env
+touch db.sqlite3 # workaround because build-dbs-for-local-development expects a db.sqlite3
+yes Y | just build-dbs-for-local-development nuclear
+
+docker-compose up -d
```

Comment:
thought: This should probably be the v2 usage:

```sh
docker compose up -d
```

The devcontainer Docker feature makes this work automatically, but we should use the more modern form.